setup.py

```diff
@@ -34,6 +34,28 @@ def mpi_info(cmd):
     return out
 
 
+def cuda_info(cmd):
+    # TODO: replace with more robust way to find CUDA toolkit
+    cuda_root = os.environ.get("CUDA_ROOT")
```

Comment:
we should address this, however

Reply:
Yes, it's [on the to-do list](https://github.com/PhilipVinc/mpi4jax/pull/29#issue-491626409)

Reply:
We don't need this if we have coda-aware MPI, right?

Reply:
In any case, copy has some interesting code
https://github.com/cupy/cupy/blob/9847ac193b44036e0adaef489abe95daa7655f92/cupy_setup_build.py#L376

Reply:
and https://github.com/cupy/cupy/blob/9847ac193b44036e0adaef489abe95daa7655f92/cupy_setup_build.py#L925

Reply:
At the moment, technically not. But if you manage to build MPI from source then it's probably not asked too much to set `CUDA_ROOT` to your CUDA installation :wink: (should be set anyway)

Reply:
Cupy is a lot more complicated because they need to find `nvcc` and the CUDA build tools. All we need is the headers and runtime library. I wouldn't over-complicate this. As a user I'd much rather do

```bash
$ CUDA_INCDIR=/.../ CUDA_LIBDIR=/.../ pip install mpi4jax
```

than debugging some magical auto-detect that stops working because my libdir is called `lib64` instead of `lib`.

But I'll see what I can come up with (or you, if you want to have a stab at it)

Reply:
Ideally I'd like CUDA_ROOT to be used if present, otherwise some smart logic to try to guess it.
I'll have a look